### ISSUE TITLE
Make right-click menu consistent

### ISSUE THREAD

**Issue by ilovan** [cwrc/CWRC-WriterBase#7]
The order of the options in the right-click menu as accessed from the structure panel differs from the order when same menu is accessed from the text area:

![screen shot 2017-11-01 at 10 40 13 am](https://user-images.githubusercontent.com/6600070/32285932-7c1d8b78-bef1-11e7-84a0-83537d2dfe39.png)
(right-click menu in text area)

-------------------------------

![screen shot 2017-11-01 at 10 39 26 am](https://user-images.githubusercontent.com/6600070/32285933-7c556e62-bef1-11e7-821e-3b83b21fb8da.png)
(right-click menu in structure area)

I think we should go with the later.

**Comment by jchartrand** [comment]
My guess is that you meant to assign this to Andrew.  I could figure it out, but Andrew probably knows exactly where to make the change.

**Comment by ilovan** [comment]
Yes, probably Andrew is the best person to deal with it, I was not sure how you guys are handling the bugs in the new code base.

**Comment by jchartrand** [comment]
I’m not sure what you mean by ‘handling the bugs’?   Is this a question about how the overall development process should flow?

**Comment by ilovan** [comment]
Just wasn't sure how the work division between you and Andrew on the GitWriter worked. From what you are saying, it looks like everything Andrew used to deal with in the old version of CWRC-Writer is still on his plate, while everything entity lookup and NPM is on yours. Please confirm

**Comment by jchartrand** [comment]
Yes.
